app/controllers/api/v1/errors_controller.rb

```diff
@@ -6,6 +6,17 @@ def index
   end
 
   def create
+    @subscriber = Subscriber.where("email = ? and website_id = ?", params['email'], @current_site.id)
+    @error = Issue.where("website_id = ? and page_title = ?", @current_site.id , params['page_title'])
+    if @subscriber.blank?
+      @subscriber = @current_site.subscribers.create( email: params['email'], name: params['email'], website_id: @current_site.id)
```

Comment:
aici schimba `@current_site` cu `current_site` si la dracu nu e nevoie de website_id pentru ca deja ai current_site in fata de aia folosesti current_site.subscribers ca sa nu mai definiesti website_id ....